app/components/support_interface/duplicate_matches_table_component.rb

```diff
@@ -1,15 +1,38 @@
 module SupportInterface
   class DuplicateMatchesTableComponent < ViewComponent::Base
     include ViewHelper
+    include Pagy::Frontend
```

Comment:
Is there any reason we are not using kaminari here and are using Pagy instead? Why do we need two different pagination libraries?

Reply:
moving this one to slack!

Reply:
Pagy was more explicit in the code than Kaminari, so when I search for any implementation of pagination in the code, I found pagy first and read all the places pagy was used, then I decided to use pagy on this PR because I wasn't aware of Kaminari neither the paginator component. 
But now that I know that we're using Kaminari (thanks to both @despo and @duncanjbrown for the heads up), I changed the implementation to use Kaminari. The PR looks much more simple! 😆 😆 😆